README.md

```diff
@@ -7,30 +7,41 @@ Open source library with libusb-based drivers to control robotic arms by Kinova.
 Installing:
```

Comment:
I'd remove this and simply put it below "Installing from precompiled packages". Still keeping hose on the second header level seems appropriate.